wire-ios/Wire-iOS/Sources/UserInterface/MainController/ZClientViewController.swift

```diff
@@ -35,7 +35,9 @@ final class ZClientViewController: UIViewController {
     let wireSplitViewController: SplitViewController = SplitViewController()
 
     private(set) var mediaPlaybackManager: MediaPlaybackManager?
+    private(set) var mainTabBarController: UITabBarController!
```

Comment:
Could it be optional?
```suggestion
    private(set) var mainTabBarController: UITabBarController?
```

Reply:
I prefer to keep it this way, since the property isn't supposed to be `nil` after `viewDidLoad` executed.

Reply:
I see and it's fine for me too.
I just want add the note that a view controller can be initialised and only when the view is accessed (e.g. adding to the view hierarchy) then `viewDidLoad` will be called, so there is a potential timing gap where it can crash.